### DISCUSSION TITLE
Disabling Google Home Announcement/Cast  "ding"

### DISCUSSION THREAD

**Discussion by tiaanv** [music-assistant/support#2187] [Feature requests and ideas]
Since I started using Google Homes in my house, I've always been plagued by the chime it plays before starting to play a stream. When doing TTS announcements from HA, this is especially irritating.  There is no setting to disable it.

A workaround does however exist.  It involves muting (setting the device volume to 0) the device first.  Unfortunately, since I want to use the advanced features of Music-assistant, I can no longer use that, as it manipulates the volume, which I like.

Is there any way that this "muting" behaviour could be baked into the Announcements Configuration as an option?

More info on the script that "masks" the chime here:
[https://community.home-assistant.io/t/disable-google-home-casting-sound/304624/34](https://community.home-assistant.io/t/disable-google-home-casting-sound/304624/34)

**Comment by marcelveldt** [comment]
You are talking about the "dong" sound cast makes itself when it starts a new stream right ? Not the announcement bell that is OPTIONAL for music assistant announcements ?

**Reply by marcelveldt** [reply]
The mute is a very hacky solution imo as you have no way to tell how long the player exactly took to accept the command etc. You would just have to guess and it will be dependent of the device, the network, the weather or whatever. We could maybe some day incorporate a setting for this but the delay time would need to be configurable (because that is just different for every situation)

**Reply by tiaanv** [reply]
Yes. It's absolutely a hack.  A horrible one.  There are some advanced versions of the script that wait until the player state changes, which adresses the timing issue.  Won't be offended if this is shot down as a bad idea 😉

**Comment by maxpd1** [comment]
Struggling with it since years. Would also be interested in it and I am wondering why there isn't an alternative firmware for Google home devices e.g. without the dong.

**Comment by X1pheR** [comment]
Just came across this post. I've also switched to using Music Assistant and would love to get rid of google its own "ding" sound before an announcement. I've already disabled the pre-announce setting within MA itself.

**Reply by OzGav** [reply]
To be clear you need to talk to google about this.

**Comment by jorbig** [comment]
+1

Would love a baked in solution or even a workaround, no matter how hacky it is.

**Reply by OzGav** [reply]
I just responded to this. Talk to google. Look at the WONTFIX label. I’m locking this discussion.
